stable/redis/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: redis
-version: 0.10.2
+version: 0.11.0
```

Comment:
Is the image change a breaking change? Should this be 1.0.0 now?

Reply:
The Redis 4.0 release notes (https://raw.githubusercontent.com/antirez/redis/4.0/00-RELEASENOTES) indicates that there have been major internal changes in the internals. IMO it is safer to bump the version for this release.

I'll bump it to version 1.0.0.